tests/test_certificates.py

```diff
@@ -29,7 +29,10 @@ def test_calculated_properties(self):
                 private_key=File(key_f, name="test.key"),
             )
 
-        self.assertEqual(datetime(2023, 2, 21, 14, 26, 51), certificate.expiry_date)
+        self.assertEqual(
+            certificate.expiry_date,
+            datetime(2023, 2, 21, 14, 26, 51, tzinfo=timezone.utc),
+        )
```

Comment:
Slight change in behaviour because now we have aware datetimes instead of naive. The original was wrong depending on the timezone/DST etc., since the UTC datetime was being interpreted as a naive datetime, which is off either 1 or 2 hours in NL.